site/content/blog/2018-04-18-version-2.md

```diff
@@ -201,4 +201,4 @@ Before, there was a `svelte.validate` method which checked your component was va
 
 ## My app is broken! Help!
 
-Hopefully this covers everything, and the update should be easier for you than it was for us. But if you find bugs, or discover things that aren't mentioned here, swing by [Gitter](https://gitter.im/sveltejs/svelte) or raise an issue on the [tracker](https://github.com/sveltejs/svelte/issues).
\ No newline at end of file
+Hopefully this covers everything, and the update should be easier for you than it was for us. But if you find bugs, or discover things that aren't mentioned here, swing by [Gitter](https://gitter.im/sveltejs/svelte) or raise an issue on the [tracker](https://github.com/sveltejs/svelte/issues).
```

Comment:
Again, I think Gitter -> Discord is a reasonable thing to change.

Reply:
done